addons/pylons/functions/fnc_showDialog.sqf

```diff
@@ -137,3 +139,12 @@ _edit ctrlAddEventHandler ["KeyDown", LINKFUNC(onNameChange)];
 
 private _checkbox = _display displayCtrl ID_CHECKBOX_MIRROR;
 _checkbox ctrlAddEventHandler ["CheckedChanged", {[(_this select 1) == 1] call FUNC(onPylonMirror)}];
+
+// PFH for distance check to vehicle
+[{
+    params ["", "_idPFH"];    
+    if ((ace_player distance GVAR(currentAircraft)) > PYLONS_DISTANCE_CHECK) exitWith {
```

Comment:
`distanceSqr`
waitUntilAndExecute?

Reply:
@dedmen What's the advantage of using `distanceSqr`?

Realised I forgot to make this exit if the dialog is closed as well.

Reply:
distance = sqrt (xDist^2+yDist^2+zDist^2)
distanceSqr = (xDist^2+yDist^2+zDist^2)
If PYLONS_DISTANCE_CHECK is known static, you can precompute a `PYLONS_DISTANCE_CHECK_SQR` constant
then just check
`(ace_player distanceSqr GVAR(currentAircraft)) > PYLONS_DISTANCE_CHECK_SQR`

you save having to do a extra square root
normally it would be a nice optimization, but in sqf the performance difference pretty small

Reply:
Ok thanks for that, is that something that just be applied in general to distance comparisons?

Reply:
It can be. You can perf profile that yourself and see if it's actually worth it.
Also you might get precision issues because the number is then twice as big. and the bigger it is the lower the precision.. but considering the engine first does `xDist^2+yDist^2+zDist^2` which produces a huge number and then does sqrt... precision shouldn't really be different.